analyzer/codechecker_analyzer/analyzers/result_handler_base.py

```diff
@@ -94,7 +94,7 @@ def analyzer_result_file(self):
                 self.buildaction.original_command
 
             out_file_name = analyzed_file_name + '_' + \
-                hashlib.md5(build_info).hexdigest() + '.plist'
+                hashlib.md5(build_info.encode()).hexdigest() + '.plist'
```

Comment:
Is it possible that a different hash will be generated with this for the same build_info?
If yes it can mess up the existing report directories a bit, because the old plist files will not be overwritten.

Reply:
I think if the string was a unicode string this should not change it at all (in Python 2). For Python 3, we need this to avoid getting an exception.

Reply:
I do not think `build_info` is a unicode string it is constructed a few lines above. There are some places with unicode strings but not everywhere.
please set the encoding to `utf8` (`.encode(encoding='UTF-8',errors='ignore')`) if for some reason the interpreter can not detect the locale settings for some reason it will fallback to ansi-c which could cause problems (analyzed file name with a non ascii character in it).

Reply:
The `utf-8` is the default if we do not set it.